src/main/resources/data.sql

```diff
@@ -3,3 +3,7 @@ INSERT INTO televisions (id, type, brand, name, price, available_size, refresh_r
 INSERT INTO televisions (id, type, brand, name, price, available_size, refresh_rate, screen_type, screen_quality, smart_tv, wifi, voice_control, hdr, bluetooth, ambi_light, original_stock, sold) VALUES (1003, '43PUS6504/12/M', 'Philips', '4K UHD LED Smart Tv', 379, 50, 60, 'LED', 'Ultra HD',  true, true, false, true, false, false, 345549, 244486);
 INSERT INTO televisions (id, type, brand, name, price, available_size, refresh_rate, screen_type, screen_quality, smart_tv, wifi, voice_control, hdr, bluetooth, ambi_light, original_stock, sold) VALUES (1004, '43PUS6504/12/S', 'Philips', '4K UHD LED Smart Tv', 379, 58, 60, 'LED', 'Ultra HD',  true, true, false, true, false, false, 6548945, 4485741);
 INSERT INTO televisions (id, type, brand, name, price, available_size, refresh_rate, screen_type, screen_quality, smart_tv, wifi, voice_control, hdr, bluetooth, ambi_light, original_stock, sold) VALUES (1005, 'OLED55C16LA', 'LG', 'LG OLED55C16LA', 989, 55, 100, 'OLED', 'ULTRA HD',  true, true, true, true, true, false, 50000, 20500);
+
+
+INSERT INTO remotecontrollers (id, compatible_with, battery_type, name, brand, price, original_stock) VALUES (100, 'Special compatible', 'AAA', 'Special remote controller', 'Special brand', 20, 5);
+INSERT INTO cimodules ( id, name, price) VALUES (10, 'testcimodule', 50), (11, 'testcimodule2', 50);
```

Comment:
Om in sql een remote controller aan tv toe te voegen: UPDATE television SET remote_controller=100 WHERE id=1003

Reply:
Let wel: dit is postgres, mysql werkt net iets anders